package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@nextbitlabs/rapido",
```

Comment:
Have you registered the account on npm?

Reply:
Yes, I created an organisation: https://www.npmjs.com/org/nextbitlabs